### ISSUE TITLE
Problem: Average Loss: Inf Average Regression Loss:Inf 

### ISSUE THREAD

**Issue by sunserforever** [qfgaohao/pytorch-ssd#94]
Hi author; I met some problems about the Loss. when the step = 300, the average Loss=Inf. Moreover, I changed the learning rate 0.01,0.001,0.00001,0.000001. All of the results, the average Loss= Inf.
2019-12-26 09:15:22  - root - INFO - Epoch: 0, Step: 100, Average Loss: 11.9684, Average Regression Loss 3.4351, Average Classification Loss: 8.5333
2019-12-26 09:17:52,386 - root - INFO - Epoch: 0, Step: 200, Average Loss: inf, Average Regression Loss inf, Average Classification Loss: 8.1938

**Comment by sunserforever** [comment]
I use the mobilenetv1-SSD model

**Comment by tianyuluan** [comment]
have you solved this problem?how to solved it? thanks!

**Comment by jasonlove521** [comment]
have you solved this problem?how to solved it? thanks!
